runtime/runtime_controller.cc

```diff
@@ -60,6 +60,7 @@ std::unique_ptr<RuntimeController> RuntimeController::Spawn(
                                           p_persistent_isolate_data,     //
                                           context_);                     //
   result->spawning_isolate_ = root_isolate_;
+  result->platform_data_.viewport_metrics.ResetForSpawn();
```

Comment:
Is it really necessary to have a special method for this? Shouldn't this work just as well?
```cpp
result->platform_data_.viewport_metrics = ViewportMetrics();
```